src/seismometer/controls/selection.py

```diff
@@ -11,26 +11,25 @@ class SelectionListWidget(ValueWidget, VBox):
 
     value = traitlets.Tuple(help="The selected values for the button list")
 
-    def __init__(self, title: str, options: tuple[str], value: tuple[str] = None, show_title: bool = True):
+    def __init__(self, options: tuple[str], *, value: tuple[str] = None, title: Optional[str] = None):
         """A vertical list of buttons for selection
 
         Parameters
         ----------
-        title : str
-            Title to be displayed above the list of buttons.
         options : tuple[str]
             Selectable options.
         value : Optional[tuple[str]], optional
             Subset of options that should be selected by default, by default None.
-        show_title : bool, optional
-            If True, shows the title, by default True.
+        title : Optional[str[], optoinal
+            Title to be displayed above the list of buttons, it not set, title not included
         """
         super().__init__()
         self.options = tuple(options)  # make immutable
         selected_values = value or ()
         selected_values = tuple(value for value in selected_values)
```

Comment:
This looks redundant, since selected_values is already a tuple.

Reply:
It could be something else like a list, while the type hint asks for a tuple, this will work with lists, I made this more clear by adopting the Iterable type hint.

Reply:
Having  `value` be an item in Iterable `value`is confusing.

Something like this would be more concise and clear

```
if value: selected_values = tuple(value)
        else: selected_values = ()
```

Reply:
Even better:

`selected_values = tuple(value) if value else ()`

Reply:
Less adamant about this one, but you could also combine the subsequent line like

`self.value = selected_values = tuple(value) if value else ()`

Reply:
FINC